src/App.js

```diff
@@ -4,16 +4,18 @@ import styled from "styled-components";
 import { nanoid } from "nanoid";
 import SearchPage from "./components/SearchPage.js";
 import { useState } from "react";
+import useLocalStorage from "./Hooks/UseLocalStorage.js";
```

Comment:
You should rename your file :) 
```suggestion
import useLocalStorage from "./Hooks/useLocalStorage.js";
```